aptos-move/move-examples/data_structures/sources/bucket_table.move

```diff
@@ -37,7 +37,7 @@ module aptos_std::bucket_table {
     }
 
     /// Create an empty BucketTable with `initial_buckets` buckets.
-    public fun new<K: drop + store, V: store>(initial_buckets: u64): BucketTable<K, V> {
+    public fun new<K: copy + store + drop, V: store>(initial_buckets: u64): BucketTable<K, V> {
```

Comment:
why not just new<K, V> here?

Reply:
As discussed, we will enforce the abilities only at creation.